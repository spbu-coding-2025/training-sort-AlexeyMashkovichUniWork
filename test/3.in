88982991 867157744 1011431418 -756891381 -593266141 -122877678 -86474537 764952967 -523251097 -788787954 532353924 -66381966 -1004677784 153719194 -210400710 873165189 542816138 513462561 -641856275 -11121953 -968184193 437182538 273375697 -697510333 -288816706 -986399347 -851253330 869254991 565251131 444442558 -712136005 -419507703 237858477 -774446411 -102657261 718334160 176417734 884610025 409545302 726908460 -977919754 -131842599 -413215330 -908855715 -1051865230 450125783 1038051297 564692732 -110153481 -677546803 -520171046 -4595851 833377558 826946474 371635639 -529180973 914288950 594124132 -733667806 405798256
