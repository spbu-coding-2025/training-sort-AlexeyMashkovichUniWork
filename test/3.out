-1051865230 -1004677784 -986399347 -977919754 -968184193 -908855715 -851253330 -788787954 -774446411 -756891381 -733667806 -712136005 -697510333 -677546803 -641856275 -593266141 -529180973 -523251097 -520171046 -419507703 -413215330 -288816706 -210400710 -131842599 -122877678 -110153481 -102657261 -86474537 -66381966 -11121953 -4595851 88982991 153719194 176417734 237858477 273375697 371635639 405798256 409545302 437182538 444442558 450125783 513462561 532353924 542816138 564692732 565251131 594124132 718334160 726908460 764952967 826946474 833377558 867157744 869254991 873165189 884610025 914288950 1011431418 1038051297
